src/CfdScript.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Cfd
+{
+  public class Script
+  {
+    private string script;
+    string[] script_items;
+
+    public Script(string script_hex)
+    {
+      script = script_hex;
+      using (ErrorHandle handle = new ErrorHandle())
+      {
+        ParseScript(handle, script_hex);
+      }
+    }
+
+    public Script(byte[] bytes)
+    {
+      script = StringUtil.FromBytes(bytes);
+      using (ErrorHandle handle = new ErrorHandle())
+      {
+        ParseScript(handle, script);
+      }
+    }
+
+    public Script(uint require_num, Pubkey[] pubkeys)
+    {
+      // FIXME not implements
```

Comment:
`throw NotImplementedException()` (or just delete/comment it out)

Reply:
👍